package.json

```
@@ -45,7 +45,7 @@
     "@endo/eslint-plugin/typescript-eslint": "^8.31.0"
   },
   "engines": {
-    "node": "^18.12 || ^20.9"
+    "node": "^18.12 || ^20.9 || ^22.14"
```

Node 22 LTS starts at 22.11 according to https://nodejs.org/en/blog/release/v22.11.0

```suggestion
    "node": "^18.12 || ^20.9 || ^22.11"
```

---

cc @usmanmani1122